setup.py

```diff
@@ -38,9 +38,7 @@ def get_version() -> str:
     "pydantic>1.1,<2.0; python_version=='3.8'",
 ]
 
-extras["torch"] = [
-    "torch",
-]
+extras["torch"] = ["torch", "safetensors"]
```

Comment:
```suggestion
extras["torch"] = [
    "safetensors",
    "torch",
]
```

(nit)

Reply:
You need to add a trailing comma `","` to the last line otherwise `ruff` will fold it indeed. Made the change in https://github.com/huggingface/huggingface_hub/pull/2033/commits/8ca8550dbfba04c2e7458239a05c81d1c1a18151.